e2e/codex-m0/supervisor.py

```
@@ -0,0 +1,148 @@
+    assert status["NoNewPrivs"].strip() == "1"
+    return {"subreaper": True, "uid": os.getuid(), "caps": 0, "nnp": True}
+
+
+def main():
+    profile = establish_profile()
+    child = os.fork()
+    if child == 0:
+        os.close(3)
+        os.close(4)
+        os.setsid()
+        os.execv(sys.argv[1], sys.argv[1:])
```

Ruff (0.16.3)

[error] Starting a process without a shell

(S606)